terraform/variables.tf

```diff
@@ -60,4 +60,9 @@ variable "git_sha" {
 }
 variable "lambda_analysis_workspace" {
   type = string
+}
+
+variable "data_lake_max_vcpus" {
+  type = number
+  default = 576
```

Comment:
Increasing the number of CPUs of the compute environment to be able to run 6*98 cores at once.
Batch requests for SPOT CPUs with compete with EMR requests, so we shouldn't go much higher to assure our geotrellis jobs will always run.